### ISSUE TITLE
DateTest & DatetimeTest are affected by the locale + Continuous Integration

### ISSUE THREAD

**Issue by carolinux** [plumdog/flask_table#34]
Hi! Some tests are affected by the default locale of the machine.

In order to run the tests on my machine (Switzerland) I needed to set the locale to the one I assume you're using - and it seems babel.dates checks the environment var LC_TIME first, so I needed to do this to get all the tests to pass:

```
export LC_TIME="en_UK.UTF-8"
# then run the tests
 nosetests -v
```

I would be up  to contribute to set up travis to do automated tests per Pull Request...

**Comment by carolinux** [comment]
whoops my bad, I didn't see this existed already.

**Comment by plumdog** [comment]
Interesting. I thought this had been fixed by #2, but perhaps LC_TIME needs to be added to the list of environment variables that are updated.

**Comment by carolinux** [comment]
ok I can add that.. it doesn't work for me if I just set the LANGUAGE
